src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -30,6 +30,16 @@ object DatabaseConversions {
     case _ => None
   }
 
+  def extractBallot(ballot: Voting.Vote): Option[String] = ballot match {
```

Comment:
This function seems redundant to me, if the `Ballot` operation always contains a `Voting.Vote` there's no need to pattern match, but we can read the vote value and wrap it in a `Some`.
Follow-up in-lined in the later comment for `convertBallot`